hs_core/management/commands/set_bag_dirty.py

```diff
@@ -0,0 +1,37 @@
+"""
```

Comment:
I'm confused why these changes are in this pull.

Reply:
I don't know how that ended up there, but I'll undo it.

Reply:
I wonder if it's because I branched off of RC-1.23... maybe submit this PR against that branch?

Reply:
heh, it was me.  I added it on accident. Removing it in another commit.